Chart.yaml

```
@@ -1,12 +1,30 @@
 apiVersion: v2
-appVersion: 1.27.0
-name: vaultwarden
+name: sample-app-name
+description: A Helm chart for Kubernetes
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
 version: 0.1.0
-description: Vaultwarden is a Bitwarden compatable server in Rust
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: "1.16.0"
 dependencies:
 - name: replicated-library
   repository: https://replicatedhq.github.io/helm-charts
   version: 0.13.3
 home: https://github.com/replicatedhq/replicated-starter-helm/charts/tree/main/
-icon: https://sesoodan.sirv.com/bitwarden_rs.png
 kubeVersion: '>=1.16.0-0'
```

```suggestion
kubeVersion: '>=1.24.0-0'
```
I'm unsure what version we should _actually_ set this too. But 1.16 is far to old. We have a lot of people on 1.19 but I think I tend toward let's keep this roughly N-2 for new vendors? They can always adjust it.

---

@chris-sanders committed suggestion.